.github/workflows/ci.yml

```
@@ -147,3 +147,22 @@ jobs:
         python3 -c "from urllib.request import urlretrieve; urlretrieve('${{ env.BUILDER_HOST }}/${{ env.BUILDER_SOURCE }}/${{ env.BUILDER_VERSION }}/builder.pyz?run=${{ env.RUN }}', 'builder')"
         chmod a+x builder
         ./builder build -p ${{ env.PACKAGE_NAME }}
+
+  openbsd:
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-22.04 # latest
```
trivial: can we pin to 22.04, which is the latest one, to keep it consistent with the other jobs. And in case, github action updates the latest image and breaks us out of blue.